code/ui/components/src/Zoom/browserSupportsCssZoom.ts

```diff
@@ -2,12 +2,11 @@ import { global } from '@storybook/global';
 
 export function browserSupportsCssZoom(): boolean {
   try {
-    const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
-    if (!isSafari) {
-      // @ts-expect-error (we're testing for browser support)
-      return global.document.implementation.createHTMLDocument('').body.style.zoom !== undefined;
-    }
-    return false;
+    // Checks if safari or firefox is being used.
+    // This check can be removed when zoom becomes standard css. Currently firefox does not support it and there is a bug in safari see here:  https://developer.mozilla.org/en-US/docs/Web/CSS/zoom#browser_compatibility
+    // regex checks if there are other bronwers because "safari" is also present when using chrome or an andriod browser whilst on mac/iphone see more here : https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent#which_part_of_the_user_agent_contains_the_information_you_are_looking_for
```

Comment:
```suggestion
    // regex checks if there are other browsers because "safari" is also present when using chrome or an android browser whilst on mac/iPhone see more here : https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent#which_part_of_the_user_agent_contains_the_information_you_are_looking_for
```